pymc3/tests/test_distributions.py

```diff
@@ -1356,13 +1356,16 @@ def test_t(self):
             lambda value, nu, mu, lam: sp.t.logcdf(value, nu, mu, lam ** -0.5),
             n_samples=10,  # relies on slow incomplete beta
         )
+        # TODO: reenable when PR #4736 is merged
+        """
```

Comment:
A reminder issue would be good. Or @ricardoV94 you rebase the other PR after we merge it and then you can enable it right there.

Reply:
I'll do that. Linking it here for now #4736